0004/README.md

```diff
@@ -0,0 +1,514 @@
+# Policy Architecture for Npm Package Provenance
+
+*   Proposer: Wietse Venema ([wietse@gmail.com](mailto:wietse@gmail.com))
+*   GitHub Issue: TODO
+*   Status: [DRAFT](../README.md#meaning-of-status-codes)
+*   Original link: [Google docs](https://docs.google.com/document/d/1ulft22Nv6OXFeVRIrVfH2bK4QcRyvv0RwDt2dhJltK4/edit?usp=sharing) (see there for reviewer comments and rejected ideas)
+
+## Summary
+
+This document proposes an architecture for policy evaluation over SLSA
+provenance, for software packages that are managed with npm, the package manager
+for Node.js. When npm installs a software package and its transitive
+dependencies, it records the installed packages in a lock file
+(package-lock.json). The information that is written to these files is a primary
+input for a proposed SLSA provenance policy evaluator.
+
+With few differences, the approach presented here is expected to generalize to
+other lock file-based software environments such as Go (go.mod and go.sum), PHP
+(composer.lock), Python (Pipfile.lock), and Ruby (Gemfile.lock).
+
+## Objectives
+
+Evaluate policies over SLSA provenance for npm software packages.
+
+Non-objectives:
+
+-  Policies for what direct or indirect package dependencies (or dependency
```

Comment:
Yes, but this still unfortunately doesn't clarify exactly what policies are in the first place.

Reply:
This concern duplicates an earlier comment. I'll resolve this one, so that we don't dilute the discussion.